cql3/Cql.g

```diff
@@ -664,6 +664,7 @@ createAggregateStatement returns [std::unique_ptr<cql3::statements::create_aggre
         std::vector<shared_ptr<cql3_type::raw>> arg_types;
         std::optional<sstring> ffunc;
         std::optional<expr::expression> ival;
+        std::optional<sstring> rfunc;
```

Comment:
Changelog should explain why we add reducefunc, not just state that we add it.

Reply:
Changed commit's message